clients/admin-ui/src/features/common/SearchBar.tsx

```diff
@@ -42,14 +42,12 @@ const SearchBar = ({
         onChange={handleSearchChange}
         {...props}
       />
-      {withClear ? (
+      {onClear ? (
```

Comment:
the datamap search bar needed a more custom `onClear` handler. this affects `SystemCatalog` which was using `withClear` but was easy enough to swap out to an `onClear`